net_infrastructure/comware_5_2.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
```

Comment:
This module looks like a fact gathering module with optional reboot, we prefer to separate these functions, fact gathering modules are also names <name>_facts .